packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2079,7 +2074,7 @@ export class ContainerRuntime
 			this.idCompressorMode === "on" ||
 			(this.idCompressorMode === "delayed" && this.connected)
 		) {
-			this._idCompressor = await this.createIdCompressor();
+			this._idCompressor = this.createIdCompressorFn();
```

Comment:
This is invoking the 'createIdCompressorFn' provided to the ctor, which is no longer async.